libs/ui/src/lib/footer/footer.module.scss

```diff
@@ -17,6 +17,7 @@
     .footerNav {
         h3 {
             color: white;
+            background-color: transparent;
```

Comment:
Jeg tror denne feilen kommer av at MainMenu i Footer animeres og fader inn når siden lastes. Så når kontrasten leses av Siteimprove og lignende så skjer det såpass tidlig at menyen fortsatt er litt gjennomsiktig.

Har en fiks på dette i feat/jr/details-poc-unified hvor jeg legger på `initial: show` i Footer for å skru av animasjonen her: https://github.com/Informasjonsforvaltning/fdk-frontend/blob/feat/jr/details-poc-unified/libs/ui/src/lib/footer/index.tsx#L25

Måtte da også utvide MainMenu med støtte for å ta imot motionProps: https://github.com/Informasjonsforvaltning/fdk-frontend/blob/feat/jr/details-poc-unified/libs/ui/src/lib/main-menu/index.tsx#L48